### ISSUE TITLE
Onboarding: Performance issues

### ISSUE THREAD

**Issue by vgrozdanic** [getsentry/sentry#77932]
I have a new account with 1 Issue. How is it possible that the Issue page takes like 3 seconds to load. That is just silly-bad performance.

OK, load times are terrible everywhere. I clicked on an issue to take me to the session replay and it was like 5+ second load time.

See if we can do some easy improvements at least to the first impression - e.g. don't fetch 14 days worth of issues if your org is 10 minutes old.

**Comment by vgrozdanic** [comment]
I don't think this is a problem for new orgs, this only seems to be a problem for larger organizations with a lot of events

**Comment by vgrozdanic** [comment]
Just did some tests, and whatever the time period is (for a newly created org) - loading time seems to be the same..

**Comment by vgrozdanic** [comment]
Closing this, since I can't repro it
https://github.com/user-attachments/assets/84351f42-a094-494c-a3c6-a627a05f7847

**Comment by vgrozdanic** [comment]
https://sentry.slack.com/archives/C04KZQBNQ2U/p1728049418058689

**Comment by mrduncan** [comment]
Going to close this to clean up our backlog as we've made a number of recent improvements in this area. If we have a clear repro case we can reopen.
